README.md

```diff
@@ -5,13 +5,70 @@ This repository is for EDGI [Web Monitoring Project](https://github.com/edgi-gov
 
 EDGI is already monitoring tens of thousands of pages and will eventually be monitoring tens of millions (or even as many as ~1 billion). Currently, there is a lot of manual labor that goes into reviewing all changes, regardless of whether they are meaningful or not. Any system will need to emphasize usability of the UI and efficiency of computational resources.
 
+- [Technologies Used](#hammer-technologies-used)
 - [Project Goals](#project-goals)
 - [How to Help](#how-to-help)
 - [Project Overview](#project-overview)
 - [License & Copyright](#license--copyright)
 
 You can **track upcoming releases** by exploring our [milestones](https://github.com/edgi-govdata-archiving/web-monitoring/milestones) within the GitHub issue queue.
 
+## :hammer: Technologies Used
+
+To help newcomers better understand where they might be able to
+contribute, these are the main tools that we use and are having
+discussions around within this project. This includes software in
+the application, but also platforms that we rely on.
+
+And don't worry -- you definitely don't need to know all of them!
+
+* **HTML & CSS.**
+* **SASS.**
+* [**Ruby on Rails.**][rails] A web application development framework used by our backend API server.
```

Comment:
@Mr0grog Can you suggest your preferred alternative to "backend API server"? Thanks!

Reply:
Ideas:

* our API server
* the server that supports the central REST API to our data and services